sdk/aqueduct/utils/function_packaging.py

```diff
@@ -74,8 +78,7 @@ def serialize_function(
             Can be either a path to the requirements.txt file or a list of pip requirements specifiers.
             (eg. ["transformers==4.21.0", "numpy==1.22.4"]. If not supplied, we'll first
             look for a `requirements.txt` file in the same directory as the decorated function
-            and install those. Otherwise, we'll attempt to infer the requirements with
-            `pip freeze`.
+            and install those. Otherwise FileNotFoundError exception will be raised.
```

Comment:
What do you think about using a custom RequirementsNotFoundError class instead? I think it may make more sense since requirements can be defined in code (@op) or in a file (requirements.txt)